firebase/deploy

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+main() {
+   set -euo pipefail
+
+   if [[ "$#" -gt 1 ]]; then
+       exit_error "usage [CHANNEL=dev]"
+   fi
+
+   channel="dev"
```

Comment:
`channel="${1:-dev}"` - set channel to `$1` if it is set, otherwise `dev`

Reply:
done. thanks.